configure.ac

```diff
@@ -305,6 +305,10 @@ AC_CONFIG_FILES([Makefile
                 pdfixed/include/Makefile
                 PI/Makefile])
 
+AS_IF([test "$want_pi" = yes], [
+  AC_CONFIG_SUBDIRS([targets/simple_switch_grpc])
```

Comment:
you should be able to remove the `targets/simple_switch_grpc/autogen.sh` file now

Reply:
Thanks! I've removed this file.